ruby3.2-version_gem.yaml

```diff
@@ -19,10 +19,12 @@ environment:
 pipeline:
   - uses: git-checkout
     with:
-      expected-commit: 05ab9c754e3945e414c85f181b9e3195cc1ec689
+      expected-commit: 5086ef73754282e1e878539819e76915bce56103
       tag: v${{package.version}}
       repository: https://gitlab.com/oauth-xx/version_gem
 
+  - runs: sed -i '/signing_key/d' ${{vars.gem}}.gemspec
```

Comment:
Thank-you for working on this. Minor suggestion - but can we add a code comment here with a quick summary why we're removing this?

Reply:
[here](https://gitlab.com/oauth-xx/version_gem/-/issues/3) is the issue maybe you could use that as a reference here.

Reply:
I just submitted a follow up PR to melange pipeline/ruby that has a description: https://github.com/chainguard-dev/melange/pull/1481

Once it get merged and updated in the Wolfi CI, we can omit similar lines and patch files (that removes signing_key) related to this. Do you think we still need a comment here? Happy to add!

Reply:
I also released a newer version of sdk and bumped the digests in Wolfi OS: https://github.com/wolfi-dev/os/pull/27924